project/dimensions/enduses.csv

```diff
@@ -0,0 +1,99 @@
+district_heating_heating,District Heating Heating,natural_gas,MBtu
+district_heating_water_systems,District Heating Water Systems,natural_gas,MBtu
+electricity_bath_fan,Bath Fan,electricity,MWh
+electricity_ceiling_fan,Ceiling Fan,electricity,MWh
+electricity_clothes_dryer,Clothes Dryer,electricity,MWh
+electricity_clothes_washer,Clothes Washer,electricity,MWh
+electricity_cooking_range,Cooking Range,electricity,MWh
+electricity_cooling,Cooling,electricity,MWh
+electricity_dishwasher,Dishwasher,electricity,MWh
+electricity_exterior_holiday_lighting,Exterior Holiday Lighting,electricity,MWh
+electricity_exterior_lighting,Exterior Lighting,electricity,MWh
+electricity_extra_refrigerator,Extra Refrigerator,electricity,MWh
+electricity_fans,Fans,electricity,MWh
+electricity_fans_cooling,Fans Cooling,electricity,MWh
+electricity_fans_heating,Fans Heating,electricity,MWh
+electricity_freezer,Freezer,electricity,MWh
+electricity_garage_lighting,Garage Lighting,electricity,MWh
+electricity_heat_recovery,Heat Recovery,electricity,MWh
+electricity_heat_rejection,Heat Rejection,electricity,MWh
+electricity_heating,Heating,electricity,MWh
+electricity_heating_supplemental,Heating Supplemental,electricity,MWh
+electricity_hot_tub_heater,Hot Tub Heater,electricity,MWh
+electricity_hot_tub_pump,Hot Tub Pump,electricity,MWh
+electricity_house_fan,House Fan,electricity,MWh
+electricity_interior_equipment,Interior Equipment,electricity,MWh
+electricity_interior_lighting,Interior Lighting,electricity,MWh
+electricity_plug_loads,Plug Loads,electricity,MWh
+electricity_pool_heater,Pool Heater,electricity,MWh
+electricity_pool_pump,Pool Pump,electricity,MWh
+electricity_pumps,Pumps,electricity,MWh
+electricity_pumps_cooling,Pumps Cooling,electricity,MWh
+electricity_pumps_heating,Pumps Heating,electricity,MWh
+electricity_range_fan,Range Fan,electricity,MWh
+electricity_recirc_pump,Recircuation Pump,electricity,MWh
+electricity_refrigeration,Refrigeration,electricity,MWh
+electricity_refrigerator,Refrigerator,electricity,MWh
+electricity_water_systems,Water Systems,electricity,MWh
+electricity_well_pump,Well Pump,electricity,MWh
+fuel_oil_heating,Heating,fuel_oil,MBtu
+fuel_oil_water_systems,Water Systems,fuel_oil,MBtu
+natural_gas_clothes_dryer,Clothes Dryer,natural_gas,MBtu
+natural_gas_cooking_range,Cooking Range,natural_gas,MBtu
+natural_gas_fireplace,Fireplace,natural_gas,MBtu
+natural_gas_grill,Grill,natural_gas,MBtu
+natural_gas_heating,Heating,natural_gas,MBtu
+natural_gas_hot_tub_heater,Hot Tub Heater,natural_gas,MBtu
+natural_gas_interior_equipment,Interior Equipment,natural_gas,MBtu
+natural_gas_lighting,Lighting,natural_gas,MBtu
+natural_gas_pool_heater,Pool Heater,natural_gas,MBtu
+natural_gas_water_systems,Water Systems,natural_gas,MBtu
+propane_clothes_dryer,Clothes Dryer,propane,MBtu
+propane_cooking_range,Cooking Range,propane,MBtu
+propane_heating,Heating,propane,MBtu
+propane_water_systems,Water Systems,propane,MBtu
+wood_heating,Heating,wood,MBtu
```

Comment:
These are the end uses dsgrid currently has for ResStock and ComStock. Are these accurate for DECARB buildings?

Reply:
Scout's end uses match those used in the AEO and are much more aggregated than those in ResStock and ComStock. You can see Scout's residential and commercial end uses [here](https://scout-bto.readthedocs.io/en/latest/ecm_reference.html#end-use). We have developed a mapping file that maps the Stock end uses to those used in Scout, but I don't think we plan to disaggregate Scout's end-use resolution back to the Stock end uses (after developing the hourly projections), so the expected data format would instead be Scout's more limited set of end uses (and only the electric ones). Can discuss if more clarification is needed.